src/pages/Home.jsx

```diff
@@ -38,6 +40,20 @@ const Home = () => {
       >
         <section id="home">
           <Stack alignItems="center" justifyContent="center">
+            <Box my={8}>
+              <Box
+                component="img"
+                src="/assets/PASSLogolightmode.png"
```

Comment:
If we go with this logo, then presumably we'll also want to update the smaller one in the navbar as well as the favicon.

Reply:
Correct, but I'm not sure this is the exact logo desired. My understanding is it is the correct design desired, but not sure on color/size/quality/etc. So, I didn't want to go too far.

Reply:
Yeah I don't think there's been a real, definitive discussion about what logo to go with